python/lsst/ts/ess/common/device_config.py

```diff
@@ -52,6 +52,12 @@ class DeviceConfig:
         Only relevant for sensors that report statistics, such as anemometers.
         For those sensors the value must be > 1, and preferably larger.
         For other sensors use the default value of 0.
+    safe_interval : `int`, optional
+        The number of seconds to sleep before an event is sent indicating that
+        a previously sent event is over.
+    threshold : `float`, optional
+        A threshold value that can be used to, for instance, determine if an
+        event needs to be sent.
```

Comment:
I think you may want to put the default values for optional arguments in doc string as the format of `numpy` doc string.